geonotebook/kernel.py

```diff
@@ -15,11 +16,16 @@
                      GeonotebookLayerCollection,
                      NoDataLayer,
                      SimpleLayer,
+                     TMSTileLayer,
                      TimeSeriesLayer,
                      VectorLayer)
 
 from .utils import get_kernel_id
-from .wrappers import RasterData, RasterDataCollection, VectorData
+from .wrappers import (RasterData,
+                       TMSRasterData,
```

Comment:
In general i'd like to try and avoid introducing new wrapper types like ```TMSRasterData```  and instead push functionality into reader's that expose the correct data/metadata through the ```RasterData``` API.  It seems like the primary reason to do this is to [test TMSRasterData](https://github.com/jpolchlo/geonotebook/blob/542d6c677c794b68f7ba0762d30650e995d82b2b/geonotebook/kernel.py#L404) in kernel.py and add a TMSTileLayer?  Is that correct? If so would you consider checking data.reader for some kind of TMSRasterDataReader class that was added through something like a [tms:// schema](https://github.com/OpenGeoscience/geonotebook/blob/master/setup.py#L300) to setup.py so it uses a different reader?